docs/swarm/services.md

```diff
@@ -0,0 +1,252 @@
+command:
+
+```
+$ docker network create --driver overlay my-network
+
+etjpu59cykrptrgw0z0hk5snf
+```
+
+After you create an overlay network in swarm mode, all manager nodes have access
+to the network.
+
+When you create a service and pass the `--network` flag to deploy the service to
+the overlay network in the swarm:
+
+```
+$ docker service create \
+  --replicas 3 \
+  --network my-multi-host-network \
+  --name my-web \
+  nginx
+
+716thylsndqma81j6kkkb5aus
+```
+
+The swarm extends the `my-network` to each node running the service.
+
+<!-- TODO when overlay-security-model is published
+For more information, refer to [Note on Docker 1.12 Overlay Network Security Model](../userguide/networking/overlay-security-model.md).-->
+
+## Configure update behavior
+
+When you create a service, you can specify a rolling update behavior for how the swarm should apply changes to the service when you run `docker service update`.
+
+The `--update-delay` flag configures the time delay between updates to a service task or sets of tasks. You can describe the time `T` as a combination of the number of seconds `Ts`, minutes `Tm`, or hours `Th`. So `10m30s` indicates a 10 minute 30 second delay.
+
+By default the scheduler updates 1 task at a time. You can pass the ``--update-parallelism` flag to configure the maximum number of service tasks that the scheduler updates simultaneously.
+
+By default, when an update to an individual task returns a state of `RUNNING`, the scheduler schedules another task to update until all tasks are updated. If, at any time during an update a task returns `FAILED`, the scheduler pauses the update. You can control the behavior using the `--update-failure-action` flag for docker service create or docker service update.
+
+In the example service below the scheduler applies updates to a maximum of 2
+replicas at a time. When an updated task returns either `RUNNING` or `FAILED`,
+the scheduler waits 10 seconds before stopping the next task to update:
+
+```bash
+$ docker service create \
+  --replicas 10 \
+  --name my_web \
+  --update-delay 10s \
+  --update-parallelism 2 \
+  --update-failure-action continue \
+  alpine
+
+0u6a4s31ybk7yw2wyvtikmu50
+```
+
+## Configure Mounts
+
+You can create two types of mounts for services in a swarm, `volume` mounts or `bind` mounts. You pass the `--mount` flag when you create a service. If you don't specify a type of mount, it defaults to volume.
+
+Volumes are persistent storage that remain once after a container for a task has been removed. The following samples show volume mount syntax:
+
+```
+# For an existing volume
+$ docker service create \
+  --mount src=<VOLUME-NAME>,dst=<CONTAINER-PATH> \
+  --name myservice \
+  <IMAGE-ID>
+
+# Same as above
+$ docker service create \
+  --mount type=volume,src=<VOLUME-NAME>,dst=<CONTAINER PATH>
+  --name myservice \
+  <IMAGE-ID>
+
+# To create a volume at service creation time
```

Comment:
It is recommend to use the volume commands, over this method. It should be used if you want the volume to created when the task is dispatched.

The volume is not created when the service is created. It is created before starting up the container associated with the service.

Reply:
> It is recommend to use the volume commands, over this method.

Why?